index.js

```diff
@@ -5222,13 +5222,24 @@ export async function submitBom(args, bomContents) {
   if (projectVersion == true) {
     projectVersion = "master";
   }
-  const bomPayload = {
-    project: args.projectId,
-    projectName: args.projectName,
-    projectVersion: projectVersion,
-    autoCreate: "true",
-    bom: encodedBomContents
-  };
+  if (typeof(args.parentProjectId) !== "undefined") {
```

Comment:
Can we avoid copy paste? We only need another argument based on this condition so the variable bomPayload can be a const

Reply:
Hi @prabhu ,

I tried with without if else but we have to support both the cases with a parent and without a parent, If we add parentUUID: args.parentProjectId in the bompayload and the user didn't pass the UUID of the parent then it will fail

Please recommend if there is a solution to avoid code duplication

Reply:
```
if (typeof(args.parentProjectId) !== "undefined") {
  bomPayload.parentUUID = args.parentProjectId;
}